tests/e2e/test_driver.py

```diff
@@ -633,6 +635,27 @@ def test_closing_a_closed_connection_doesnt_fail(self):
             
             self.assertTrue(expected_message_was_found, "Did not find expected log messages")
 
+    @skipUnless(pysql_supports_arrow(), 'needs arrow support')
+    def test_cloud_fetch(self):
+        # This test can take several minutes to run
+        limits = [100000, 300000]
+        threads = [10, 25]
+        self.buffer_size_bytes = 104857600
+        self.arraysize = 100000
+        base_query = "SELECT * FROM store_sales WHERE ss_sold_date_sk = 2452234 "
```

Comment:
This appears to require the `samples` catalog. Where are you setting that in this code?

Normally we run our tests against a catalog called `main` and the schema varies from one run to the next?

Reply:
I think it's actually catalog `hive_metastore` that has the store_sales table. Tbh, any large table with lots of rows would do and this is the one I found. Is it okay to specify a different catalog from `main` and only test against the `default` schema (as I'm doing in the latest commit)?

Reply:
That should be fine, but I would document in the comments the reason for this.